tools/lsp/preview.rs

```diff
@@ -134,6 +134,11 @@ fn search_for_parent_element(root: &ElementRc, child: &ElementRc) -> Option<Elem
     None
 }
 
+// triggered from the UI, running in UI thread
```

Comment:
Not for this PR, but the need for these comment worries me, and It would be better to use a different module for the code that runs in the ui thread, and the code that doesn't.